src/scenario_tests.rs

```diff
@@ -1271,6 +1271,31 @@ mod src_tests {
         );
     }
 
+    #[test]
+    fn parsec_check_work_unit_increment_has_no_effect_if_relocating_node() {
+        let initial_state = arrange_initial_state(
+            &initial_state_old_elders(),
+            &[TestEvent::SetWorkUnitEnoughToRelocate(YOUNG_ADULT_205).to_event()],
+        );
+
+        run_test(
+            "Additional WorkUnitIncrement does not trigger a new relocate if one started",
+            &initial_state,
+            &[
+                ParsecVote::WorkUnitIncrement.to_event(),
+                TestEvent::SetWorkUnitEnoughToRelocate(NODE_ELDER_130).to_event(),
+                ParsecVote::WorkUnitIncrement.to_event(),
```

Comment:
You probably could have kept these 2 in the arrange_initial_state.

```
                ParsecVote::WorkUnitIncrement.to_event(),
                TestEvent::SetWorkUnitEnoughToRelocate(NODE_ELDER_130).to_event(),
```

What we test is only the last one

Reply:
Yeah, I did have that originally, but I forgot we purge the events accumulated before we call `run_test()` though, so it makes the state we're asserting on not as useful.  We'd need to add more events presumably to check we have still got the right candidate, or dive into the member nodes of `action` to check which one(s) are set as relocating.

Reply:
The way I see test in genera, is, we have another test, that I think already exists, that tell us what we expect for the previous event. here we are just testing the last event.

Reply:
I agree in general.  But in this case that's insufficient.  If I moved the 2 suggested events into the initial state, then the assert would simply be:

```rust
&AssertState {
    action_our_events: vec![
        ActionTriggered::WorkUnitIncremented.to_event(),
    ],
},
```

Maybe I'm not following what you mean, but I don't think that's enough to ensure that the second `WorkUnitIncrement` didn't overwrite the adult with the elder as the relocating node, or marking them both for relocation.

Reply:
I understand your point now and have pushed another fixup.  Thanks :)